### ISSUE TITLE
Extend to support more things

### ISSUE THREAD

**Issue by Iaotle** [NightmareXIV/SelectString#2]
I know you're prolly busy with 6.1, but have you thought about extending this to work with more things?

Retainers, aetheryte selection, etc

**Comment by Limiana** [comment]
Yeah, that's how it was planned initially, however as a right now I'm having not much spare time at all. Hopefully, I will have more soon though

**Comment by coredumperror** [comment]
I hate to bump an old thread, but I basically do want to ask the same question again. It's 2 years later, and I'm wondering if you'd have the time or inclination, now, to add support for more things?

I'd *especially* love for this to work with the price confirmation dialog on marketboard sales. Being able to press "1" instead of moving my mouse to the Confirm button for every single potion stack I want to re-price would be amazing.

**Comment by Limiana** [comment]
Yeah, it's a good question, I think I will be trying to revive this plugin in Dawntrail, I should have time for it hopefully
As for marketboard, you could use marketbuddy plugin https://github.com/chalkos/Marketbuddy

**Comment by coredumperror** [comment]
I actually did discover the power of MarketBuddy shortly after making this comment. I met someone who showed me how to properly use the "Hold Ctrl" functionality, which I hadn't understood before, and it completely solves my issue.

That said, there's still plenty of other menus and dialogs that could use some love from this plugin. The Lower Quality confirmation dialog, for instance (I don't want it to auto-accept, like some tweaks enable; I want it to be easier to confirm manually). And the Housing menu.
